VoodooI2CHID/VoodooI2CMultitouchHIDEventDriver.cpp

```diff
@@ -431,7 +431,7 @@ bool VoodooI2CMultitouchHIDEventDriver::handleStart(IOService* provider) {
     if (!digitiser.transducers)
         return false;
 
-    if (parseElements() != kIOReturnSuccess) {
+    if (parseElements(0) != kIOReturnSuccess) {
```

Comment:
What does `0` imply here, it's more or less a magic constant? Can you use a constant from IO Kit?